error-handling-core/src/test/java/com/bakdata/kafka/ErrorCapturingFlatKeyValueMapperTopologyTest.java

```diff
@@ -73,8 +74,9 @@ protected void buildTopology(final StreamsBuilder builder) {
         mapped.flatMapValues(ProcessedKeyValue::getValues)
                 .to(OUTPUT_TOPIC, Produced.with(DOUBLE_SERDE, LONG_SERDE));
         mapped.flatMap(ProcessedKeyValue::getErrors)
-                .transformValues(DeadLetterTransformer.createDeadLetter("Description"))
-                .to(ERROR_TOPIC);
+                .transformValues(
+                        DeadLetterTransformer.create("Description", deadLetterDescription -> deadLetterDescription))
+                .to(ERROR_TOPIC, Produced.valueSerde(DEAD_LETTER_SERDE));
```

Comment:
why not configure the default value serde properly?

Reply:
ah, makes sense. Thought this referred to the proto-tests....